### PR TITLE
Goodbye Tuinity

### PR DESCRIPTION
**Pull request by Encode42**
Tuinity will be merged with Paper in the near future, this PR should take care of that mismatch

### CODE REVIEW

**Comment by YouHaveTrouble** [comment]
Will merge when we'll get an officil stable build of paper with tuinity patches

**Comment by Trevo525** [comment]
Is this the only source for this? https://github.com/PaperMC/Paper/commit/ba4cf68774472754159d45ea71f3273beb5743fd

I was hoping to read more about this merge. I figured there would be a blog post about it. But, all I can find is this commit that says tuinity in the message.

**Comment by Encode42** [comment]
https://github.com/PaperMC/Paper/pull/6413
+ Paper and Tuinity's announcements in their respective Discord servers.

**Comment by Encode42** [comment]
yes

File: README.md [hunk 1]
```diff
@@ -18,7 +18,6 @@ Your choice of server software can make a huge difference in performance and api
 
 Recommended top picks:
 * [Paper](https://github.com/PaperMC/Paper) - The most popular server software that aims to improve performance while fixing gameplay and mechanics inconsistencies.
-* [Tuinity](https://github.com/Spottedleaf/Tuinity) - Paper fork that includes even more high-performance patches.
 * [Airplane](https://github.com/Technove/Airplane) - Tuinity fork that aims to further improve server performance.
```
Comment (YouHaveTrouble) [inline README.md:21]: how come tuinity for is real when tuinity is not real

Comment (Encode42) [reply README.md:21]: because merge conflicts

**Pull request merged**